packages/flutter_tools/lib/src/commands/create.dart

```diff
@@ -39,7 +39,12 @@ const List<String> _kAvailablePlatforms = <String>[
 ];
 
 const String _kNoPlatformsErrorMessage = '''
-The plugin project was generated without specifying the `--platforms` flag, no new platforms are added.
+The plugin project does not seem to support any platform.
+
+This determination is based on the folders and files in the project directory assuming best practise of naming
```

Comment:
I'm not sure what this means - I don't think we support configuring the platform folders

Reply:
This message is mainly for web.
Because there is no "web" folder, the only way we determine if the plugin supports web is to check if `lib/plugin_web.dart` exists.
However, people can edit that file to a different name. So we have to say something about it like "this claim is not deterministic if the plugin already supports web"

Any suggestion on the message? :) 

How about:

If the plugin currently only supports web, you might see this message because the web file name was not ending with "web.dart". You can discard this message if that was the case.

Reply:
but the web still needs a manifest entry

Reply:
I think you should reach out to the UXR team for help with this one.